PWSH Post-SetContext Original Status Variable is One Result Behind

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

## Description
Result from `[bool]$originalStatus` in `Posh-SetContext` is one result behind where it should be.

## Solution
```
Move https://github.com/JanDeDobbeleer/oh-my-posh/blob/00c3e196bbb95bcaf167dcf6455a33850dac83ad/src/shell/scripts/omp.ps1#L445-L447 above https://github.com/JanDeDobbeleer/oh-my-posh/blob/00c3e196bbb95bcaf167dcf6455a33850dac83ad/src/shell/scripts/omp.ps1#L443
```

Edit: moving those two around won't fix the problem because the if statement around `Update-PoshErrorCode` requires a value that is set in `Set-PoshPromptType`. Instead, move `Set-PoshContext $script:ErrorCode` outside of `Set-PoshPromptType` and place it after `Update-PoshErrorCode`.

## Notes
I'm going to go poke around at this and test it to make sure moving those lines of code around don't break anything, but hopefully it's a simple fix.

### Theme

clean-detailed with a single added text segment.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Shell(true)                                -   0 ms
Sysinfo(true)                              -   0 ms
Executiontime(true)                        -   0 ms
Git(false)                                 -  13 ms
Text(true)                                 -   0 ms
Time(true)                                 -   0 ms
Path(true)                                 -   1 ms
Text(true)                                 -   0 ms
Status(true)                               -   0 ms

Run duration: 19.6812ms

Cache path: C:\Users\Soyvolon\AppData\Local\oh-my-posh

Config path: C:\Users\Soyvolon\OneDrive\Documents\oh-my-themes\clean-detailed-mommy.omp.json

Logs:

[DEBUG] 16:15:29.071 debug.go:42 → debug mode enabled
[DEBUG] 16:15:29.071 debug.go:42 → dlain mode enabled
[DEBUG] 16:15:29.071 os.go:Getenv:333 → C:\Users\Soyvolon\AppData\Local
[TRACE] 16:15:29.071 os.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 16:15:29.071 os_windows.go:CachePath() - 0s
[DEBUG] 16:15:29.071 os.go:238 → loading cache file: C:\Users\Soyvolon\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 16:15:29.072 os.go:238 → loading cache key: prompt_count_cache_14784
[DEBUG] 16:15:29.072 os.go:238 → loading cache key: prompt_count_cache_17284
[DEBUG] 16:15:29.072 os.go:238 → loading cache key: template_cache_14784
[DEBUG] 16:15:29.072 os.go:238 → loading cache key: template_cache_17284
[DEBUG] 16:15:29.072 os.go:238 → loading cache key: upgrade_check
[TRACE] 16:15:29.072 os.go(C:\Users\Soyvolon\AppData\Local\oh-my-posh\omp.cache) - 588.3µs
[DEBUG] 16:15:29.072 os.go:Getenv:333 → C:\Users\Soyvolon\AppData\Local
[TRACE] 16:15:29.072 os.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 16:15:29.072 os_windows.go:CachePath() - 0s
[DEBUG] 16:15:29.072 os.go:238 → loading cache file: C:\Users\Soyvolon\AppData\Local\oh-my-posh\omp.cache.23572
[DEBUG] 16:15:29.073 os.go:238 → loading cache key: prompt_count_cache
[DEBUG] 16:15:29.073 os.go:238 → loading cache key: template_cache
[TRACE] 16:15:29.073 os.go(C:\Users\Soyvolon\AppData\Local\oh-my-posh\omp.cache.23572) - 504.3µs
[DEBUG] 16:15:29.073 os.go:Getenv:333 → C:\Users\Soyvolon\OneDrive\Documents\oh-my-themes\clean-detailed-mommy.omp.json
[TRACE] 16:15:29.073 os.go:Getenv(POSH_THEME) - 0s
[DEBUG] 16:15:29.073 os.go:resolveConfigPath:261 → config set using POSH_THEME: C:\Users\Soyvolon\OneDrive\Documents\oh-my-themes\clean-detailed-mommy.omp.json
[TRACE] 16:15:29.073 os.go:resolveConfigPath() - 0s
[TRACE] 16:15:29.073 os.go:SetPromptCount() - 0s
[TRACE] 16:15:29.073 os.go:Init() - 1.0926ms
[TRACE] 16:15:29.073 os.go:Flags() - 0s
[TRACE] 16:15:29.073 load.go:loadConfig() - 590.8µs
[TRACE] 16:15:29.073 os.go:Flags() - 0s
[DEBUG] 16:15:29.073 debug.go:50 → terminal program: Windows Terminal
[DEBUG] 16:15:29.073 debug.go:50 → terminal shell: shell
[DEBUG] 16:15:29.074 os.go:Getenv:333 → NO DATA
[TRACE] 16:15:29.074 os.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 16:15:29.074 os_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 os_windows.go:WindowsRegistryKeyValue:226 → ColorizationColor(DWORD): 0xC40078D4
[TRACE] 16:15:29.074 os.go:Shell() - 0s
[DEBUG] 16:15:29.074 os.go:Getenv:333 → 7.4.4
[TRACE] 16:15:29.074 os.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 16:15:29.074 debug.go:PrintDebug:23 → segment: Title
[DEBUG] 16:15:29.074 text.go:Render:79 → rendering template: {{ .Folder }}
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[TRACE] 16:15:29.074 os_windows.go:Root() - 0s
[TRACE] 16:15:29.074 os.go:Shell() - 0s
[TRACE] 16:15:29.074 os.go:StatusCodes() - 0s
[TRACE] 16:15:29.074 os_windows.go:IsWsl() - 0s
[DEBUG] 16:15:29.074 os.go:TemplateCache:822 → environment: [ALLUSERSPROFILE=C:\ProgramData AMDRMPATH=C:\Program Files\AMD\RyzenMaster\ ANDROID_AVD_HOME=F:\Virtual Machines APPDATA=C:\Users\Soyvolon\AppData\Roaming asl.log=Destination=file ChocolateyInstall=C:\ProgramData\chocolatey ChocolateyLastPathUpdate=132964948957418756 CommonProgramFiles=C:\Program Files (x86)\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=DESKTOP-QMO7328 ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DriverData=C:\Windows\System32\Drivers\DriverData GIT_LFS_PATH=C:\Program Files\Git LFS HOMEDRIVE=C: HOMEPATH=\Users\Soyvolon LOCALAPPDATA=C:\Users\Soyvolon\AppData\Local LOGONSERVER=\\DESKTOP-QMO7328 MOMMY=mommy is so proud of you~ NUMBER_OF_PROCESSORS=12 OculusBase=F:\Oculus\ OS=Windows_NT Path=C:\Program Files\PowerShell\7;F:\Oculus\Support\oculus-runtime;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git LFS;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\ProgramData\chocolatey\bin;C:\Users\Soyvolon\AppData\Local\bin\NASM;C:\Program Files (x86)\Mikero\DePboTools\bin;C:\Program Files\nodejs\;C:\Program Files\GitHub CLI\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files\Calibre2\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\1Password CLI;C:\Program Files\dotnet\;C:\Strawberry\perl\bin;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Program Files\Tailscale\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\Docker\Docker\resources\bin;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\PowerShell\7\;C:\Python310\Scripts\;C:\Python310\;C:\Users\Soyvolon\AppData\Local\Microsoft\WindowsApps;C:\Users\Soyvolon\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Soyvolon\AppData\Local\GitHubDesktop\bin;C:\Program Files\ffmpeg\bin;C:\Program Files (x86)\Mikero\DePboTools\bin;C:\Users\Soyvolon\AppData\Roaming\npm;E:\SteamLibrary\steamapps\common\Arma 3 Tools\Publisher;C:\Program Files\7-Zip;C:\Users\Soyvolon\.dotnet\tools;C:\Program Files\Modrinth;C:\Users\Soyvolon\AppData\Local\Programs\MiKTeX\miktex\bin\x64\;C:\Users\Soyvolon\AppData\Local\Pandoc\;C:\Program Files\Graphviz\bin;C:\Users\Soyvolon\full-sail-bats;C:\Program Files\android-platform-tools_r34.0.5-windows\platform-tools;C:\Users\Soyvolon\ahk\bin;C:\Users\Soyvolon\ahk;C:\Program Files\qpdf 11.9.0\bin;C:\Program Files (x86)\GnuWin32\bin;C:\Users\Soyvolon\AppData\Local\Programs\oh-my-posh\bin;C:\Users\Soyvolon\source\repos\mommy\build\bin;D:\MikuMiku\DatabaseConverter;D:\MikuMiku\FarcPack;D:\MikuMiku\MikuMikuModel;D:\MikuMiku\noesisv4474;C:\Users\Soyvolon\source\repos\KCArcadeModelExtractor\KCArcadeModelExtractor\bin\Release\net8.0\publish; PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=27 POSH_INSTALLER=manual POSH_PID=23572 POSH_SHELL_VERSION=7.4.4 POSH_THEME=C:\Users\Soyvolon\OneDrive\Documents\oh-my-themes\clean-detailed-mommy.omp.json POSH_THEMES_PATH=C:\Users\Soyvolon\AppData\Local\Programs\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Pro PROCESSOR_ARCHITECTURE=x86 PROCESSOR_ARCHITEW6432=AMD64 PROCESSOR_IDENTIFIER=AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD PROCESSOR_LEVEL=23 PROCESSOR_REVISION=7100 ProgramData=C:\ProgramData ProgramFiles=C:\Program Files (x86) ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\Soyvolon\OneDrive\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\Soyvolon\AppData\Local\Temp TMP=C:\Users\Soyvolon\AppData\Local\Temp USERDOMAIN=DESKTOP-QMO7328 USERDOMAIN_ROAMINGPROFILE=DESKTOP-QMO7328 USERNAME=Soyvolon USERPROFILE=C:\Users\Soyvolon VBOX_HWVIRTEX_IGNORE_SVM_IN_USE=1 VBOX_MSI_INSTALL_PATH=C:\Program Files\Oracle\VirtualBox\ windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=8533eade-80e1-4604-acd4-46ba1120ebcd]
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 os.go:Pwd:363 → C:\Users\Soyvolon
[TRACE] 16:15:29.074 os.go:Pwd() - 512.1µs
[DEBUG] 16:15:29.074 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.074 os_windows.go:IsWsl() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 os.go:User:501 → Soyvolon
[TRACE] 16:15:29.074 os.go:User() - 0s
[DEBUG] 16:15:29.074 os.go:Host:518 → DESKTOP-QMO7328
[TRACE] 16:15:29.074 os.go:Host() - 0s
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 os.go:Getenv:333 → NO DATA
[TRACE] 16:15:29.074 os.go:Getenv(SHLVL) - 0s
[TRACE] 16:15:29.074 os.go:TemplateCache() - 512.1µs
[TRACE] 16:15:29.074 os.go:Shell() - 0s
[DEBUG] 16:15:29.074 os.go:Getenv:333 → 27
[TRACE] 16:15:29.074 os.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 16:15:29.074 os.go:Getenv:333 → 1
[TRACE] 16:15:29.074 os.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[TRACE] 16:15:29.074 os.go:Pwd() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[DEBUG] 16:15:29.074 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 segment.go:SetEnabled:115 → segment: Executiontime
[TRACE] 16:15:29.074 os.go:Pwd() - 0s
[DEBUG] 16:15:29.074 executiontime.go:Enabled:57 → always_enabled: false
[TRACE] 16:15:29.074 os.go:ExecutionTime() - 0s
[DEBUG] 16:15:29.074 executiontime.go:Enabled:59 → threshold: 0.000000
[DEBUG] 16:15:29.074 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 executiontime.go:Enabled:63 → roundrock
[DEBUG] 16:15:29.074 segment.go:SetEnabled:115 → segment: Shell
[TRACE] 16:15:29.074 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[DEBUG] 16:15:29.074 shell.go:Enabled:28 → mapped_shell_names: map[]
[TRACE] 16:15:29.074 os.go:Shell() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[TRACE] 16:15:29.074 os.go:Pwd() - 0s
[TRACE] 16:15:29.074 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.074 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 segment.go:SetEnabled:115 → segment: Os
[TRACE] 16:15:29.074 os.go:GOOS() - 0s
[DEBUG] 16:15:29.074 os.go:Enabled:34 → 
[TRACE] 16:15:29.074 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.074 os.go:Flags() - 0s
[DEBUG] 16:15:29.074 sysinfo.go:Init:36 → precision: 2
[TRACE] 16:15:29.075 os.go:Pwd() - 0s
[DEBUG] 16:15:29.075 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.075 os.go:GOOS() - 0s
[DEBUG] 16:15:29.075 segment.go:SetEnabled:115 → segment: Sysinfo
[TRACE] 16:15:29.075 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.075 text.go:Render:79 → rendering template:  {{ .FormattedMs }}
[TRACE] 16:15:29.075 os.go:Flags() - 0s
[DEBUG] 16:15:29.075 text.go:Render:79 → rendering template:  MEM: {{ round .PhysicalPercentUsed .Precision }}% | {{ (div ((sub .PhysicalTotalMemory .PhysicalAvailableMemory)|float64) 1073741824.0) }}/{{ (div .PhysicalTotalMemory 1073741824.0) }}GB 
[TRACE] 16:15:29.075 os.go:Flags() - 0s
[DEBUG] 16:15:29.075 text.go:Render:79 → rendering template:  {{ .Name }}
[TRACE] 16:15:29.075 os.go:Flags() - 0s
[DEBUG] 16:15:29.075 text.go:Render:79 → rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}}
[TRACE] 16:15:29.075 os.go:Flags() - 0s
[TRACE] 16:15:29.075 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.075 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.076 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.076 os.go:Shell() - 0s
[DEBUG] 16:15:29.076 text.go:Render:79 → rendering template: diamond
[TRACE] 16:15:29.076 os.go:Flags() - 0s
[DEBUG] 16:15:29.076 text.go:Render:79 → rendering template: diamond
[TRACE] 16:15:29.076 os.go:Flags() - 0s
[DEBUG] 16:15:29.076 text.go:Render:79 → rendering template: diamond
[TRACE] 16:15:29.076 os.go:Flags() - 0s
[DEBUG] 16:15:29.076 text.go:Render:79 → rendering template: diamond
[TRACE] 16:15:29.076 os.go:Flags() - 0s
[TRACE] 16:15:29.076 os.go:Shell() - 0s
[TRACE] 16:15:29.076 os.go:Shell() - 0s
[TRACE] 16:15:29.076 os.go:Flags() - 0s
[TRACE] 16:15:29.076 os.go:Pwd() - 0s
[DEBUG] 16:15:29.076 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.076 os.go:GOOS() - 0s
[DEBUG] 16:15:29.076 segment.go:SetEnabled:115 → segment: Git
[TRACE] 16:15:29.076 os.go:GOOS() - 0s
[DEBUG] 16:15:29.076 os.go:Getenv:333 → C:\Users\Soyvolon\AppData\Local
[TRACE] 16:15:29.076 os.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 16:15:29.089 os.go:CommandPath:560 → C:\Program Files\Git\cmd\git.exe
[TRACE] 16:15:29.089 os.go:CommandPath(git.exe) - 13.1633ms
[TRACE] 16:15:29.089 os.go:HasCommand(git.exe) - 13.1633ms
[DEBUG] 16:15:29.089 git.go:shouldDisplay:285 → fetch_bare_info: false
[TRACE] 16:15:29.089 os.go:Pwd() - 0s
[DEBUG] 16:15:29.089 os.go:ResolveSymlink:460 → C:\Users\Soyvolon
[TRACE] 16:15:29.089 os.go:ResolveSymlink(C:\Users\Soyvolon) - 0s
[ERROR] 16:15:29.090 os.go:HasParentFilePath:724 → CreateFile .git: The system cannot find the file specified.
[TRACE] 16:15:29.090 os.go:HasParentFilePath(.git) - 526.7µs
[TRACE] 16:15:29.090 os.go:Shell() - 0s
[TRACE] 16:15:29.090 os.go:Shell() - 0s
[TRACE] 16:15:29.090 os.go:Flags() - 0s
[TRACE] 16:15:29.090 os.go:Flags() - 0s
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[TRACE] 16:15:29.090 os.go:Flags() - 0s
[DEBUG] 16:15:29.090 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:Flags() - 0s
[DEBUG] 16:15:29.090 os_windows.go:58 → C:\Users\Soyvolon
[DEBUG] 16:15:29.090 segment.go:SetEnabled:115 → segment: Path
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[DEBUG] 16:15:29.090 segment.go:SetEnabled:115 → segment: Time
[DEBUG] 16:15:29.090 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[DEBUG] 16:15:29.090 time.go:Enabled:29 → 15:04
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[DEBUG] 16:15:29.090 segment.go:SetEnabled:115 → segment: Text
[TRACE] 16:15:29.090 os.go:PathSeparator() - 0s
[TRACE] 16:15:29.090 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.090 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[TRACE] 16:15:29.090 os.go:Shell() - 0s
[DEBUG] 16:15:29.090 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.090 os.go:Shell() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:Flags() - 0s
[DEBUG] 16:15:29.090 segment.go:SetEnabled:115 → segment: Root
[DEBUG] 16:15:29.090 path.go:replaceMappedLocations:516 → mapped_locations_enabled: true
[DEBUG] 16:15:29.090 path.go:replaceMappedLocations:517 → 
[DEBUG] 16:15:29.090 path.go:replaceMappedLocations:518 → 
[DEBUG] 16:15:29.090 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os_windows.go:Root() - 0s
[DEBUG] 16:15:29.090 path.go:replaceMappedLocations:519 → 
[DEBUG] 16:15:29.090 path.go:replaceMappedLocations:524 → mapped_locations: map[]
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[TRACE] 16:15:29.090 os.go:GOOS() - 0s
[DEBUG] 16:15:29.090 path.go:makeFolderFormatMap:840 → NO DATA
[TRACE] 16:15:29.090 os.go:Pwd() - 0s
[TRACE] 16:15:29.090 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.090 os.go:StackCount() - 0s
[DEBUG] 16:15:29.091 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 16:15:29.091 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 16:15:29.091 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 16:15:29.091 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 16:15:29.091 os_windows.go:DirIsWritable() - 504.5µs
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template:  {{ .Path }}
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: ╭─
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template:  ♥ {{ .CurrentDate | date .Format }} |
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:Pwd() - 0s
[DEBUG] 16:15:29.091 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.091 os.go:GOOS() - 0s
[DEBUG] 16:15:29.091 segment.go:SetEnabled:115 → segment: Text
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: {{ .Env.MOMMY }}
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 os_windows.go:TerminalWidth:116 → terminal width: 120
[TRACE] 16:15:29.091 os_windows.go:TerminalWidth() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[DEBUG] 16:15:29.091 status.go:Init:53 → {{ .Code }}
[TRACE] 16:15:29.091 os.go:Pwd() - 0s
[DEBUG] 16:15:29.091 os_windows.go:58 → C:\Users\Soyvolon
[TRACE] 16:15:29.091 os.go:GOOS() - 0s
[DEBUG] 16:15:29.091 segment.go:SetEnabled:115 → segment: Status
[TRACE] 16:15:29.091 os.go:StatusCodes() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: {{ .Code }}
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.091 status.go:Enabled:42 → always_enabled: true
[TRACE] 16:15:29.091 os.go:TemplateCache() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: ╰─
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[DEBUG] 16:15:29.091 text.go:Render:79 → rendering template: plain
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[TRACE] 16:15:29.091 os.go:Flags() - 0s
[TRACE] 16:15:29.091 os.go:Shell() - 0s
[DEBUG] 16:15:29.091 os.go:Getenv:333 → C:\Users\Soyvolon\AppData\Local
[TRACE] 16:15:29.091 os.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 16:15:29.092 os_windows.go:CachePath() - 522.9µs
[TRACE] 16:15:29.092 os.go:Flags() - 0s
```

--- a/src/shell/scripts/omp.ps1
+++ b/src/shell/scripts/omp.ps1
@@ -387,8 +387,6 @@ Example:
             }
             catch {}
         }
-
-        Set-PoshContext $script:ErrorCode
     }
 
     function Update-PoshErrorCode {
@@ -445,6 +443,8 @@ Example:
         if ($script:PromptType -ne 'transient') {
             Update-PoshErrorCode
         }
+
+        Set-PoshContext $script:ErrorCode
 
         $cleanPSWD = Get-CleanPSWD
         $stackCount = global:Get-PoshStackCount

